examples/Example4_ChangeAddress/Example4_ChangeAddress.ino

```diff
@@ -28,6 +28,9 @@ uint8_t deviceAddress = kQwiicUltrasonicDefaultAddress; // 0x2F
 
 // New addres is 7-bit unshifted.
 uint8_t NEW_ADDR = 0x1E;
+//If using an older version of the Qwiic Ultrasonic, your address range is: 0x20 - 0x2F
+//uint8_t NEW_ADDR = 0x2F;
+
```

Comment:
Use camelcase here.

Also, I'd suggest having the default new address in this example be a value that works with both firmware versions (eg. `0x20`), and just have a comment saying something like "new firmware accepts any valid I2C address, old firmware is 0x20 to 0x2F". Then there's no need for the commented out code.